android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/storage/Migration.kt

```diff
@@ -0,0 +1,18 @@
+package covidsafepaths.bt.exposurenotifications.storage
+
+import io.realm.DynamicRealm
+import io.realm.FieldAttribute
+import io.realm.RealmMigration
+
+internal class Migration: RealmMigration {
```

Comment:
Will this work with the `KeyValues.kt` class?

Reply:
That class was already added before this commit, I'm not sure if it's migration was handled properly.
But it shouldn't have any impact if we are not in production (users can uninstall the app and install it again to fix migration errors)